### PR TITLE
condition added

### PR DESCRIPTION
**Pull request by gayathrim997**

### CODE REVIEW

**Review by bharatjarvis — APPROVED** [review]

File: CoinDock-api/app/Http/Controllers/V1/UserController.php [hunk 1]
```diff
@@ -74,11 +74,15 @@ public function primaryCurrency(User $user)
   }
   public function topPerformer(User $user)
   {
-    if ($user->wallets->isEmpty()) {
+    if($user->wallets->count() == 1){
+      return Null;
```
Comment (sita7-debug) [inline CoinDock-api/app/Http/Controllers/V1/UserController.php]: returning Null? 

it doesn't mean response for an end-point. We follow some formats to handle responses, please use them accordingly.

File: CoinDock-api/app/Http/Controllers/V1/UserController.php [hunk 2]
```diff
@@ -95,7 +99,10 @@ public function topPerformer(User $user)
 
   public function lowPerformer(User $user)
   {
-    if ($user->wallets->isEmpty()) {
+    if($user->wallets->count() == 1){
+      return Null;
```
Comment (sita7-debug) [inline CoinDock-api/app/Http/Controllers/V1/UserController.php]: same here

**Review by sita7-debug — CHANGES_REQUESTED** [review]

File: CoinDock-api/app/Http/Controllers/V1/UserController.php (hunk 1 above)
Comment (gayathrim997) [reply CoinDock-api/app/Http/Controllers/V1/UserController.php]: done

File: CoinDock-api/app/Http/Controllers/V1/UserController.php (hunk 2 above)
Comment (gayathrim997) [reply CoinDock-api/app/Http/Controllers/V1/UserController.php]: done

File: CoinDock-api/app/Http/Controllers/V1/UserController.php [hunk 3]
```diff
@@ -66,35 +66,46 @@ public function primaryCurrency(User $user)
...
+        'coin_name' => $totalPrimaryCurrency['coin_name'],
+        'balance' => $totalPrimaryCurrency['balance']
       ]
     ], Response::HTTP_OK);
   }
   public function topPerformer(User $user)
   {
+    if ($user->wallets->count() == 1) {
+      return response([
+        'message' => 'Null'
```
Comment (sita7-debug) [inline CoinDock-api/app/Http/Controllers/V1/UserController.php]: Instead of passing Null as a message, please add a sentence.

**Review by sita7-debug — CHANGES_REQUESTED** [review]

**Review by sita7-debug — APPROVED** [review]

**Pull request merged**